pymongo/monitor.py

```diff
@@ -316,7 +317,9 @@ def _get_seedlist(self):
         Returns a list of ServerDescriptions.
         """
         try:
-            seedlist, ttl = _SrvResolver(self._fqdn).get_hosts_and_min_ttl()
+            seedlist, ttl = _SrvResolver(self._fqdn,
+                                         srv_service_name=self._srv_service_name
+                                         ).get_hosts_and_min_ttl()
```

Comment:
The style here would be cleaner like this:
```python
        try:
            resolver = _SrvResolver(
                self._fqdn, 
                srv_service_name=self._srv_service_name)
            seedlist, ttl = resolver.get_hosts_and_min_ttl()
```

Reply:
done.